llama-index-integrations/vector_stores/llama-index-vector-stores-relyt/llama_index/vector_stores/relyt/base.py

```diff
@@ -59,9 +59,10 @@ class RelytVectorStore(BasePydanticVectorStore):
     _client: "PGVectoRs" = PrivateAttr()
     _collection_name: str = PrivateAttr()
 
-    def __init__(self, client: "PGVectoRs", collection_name: str) -> None:
+    def __init__(self, client: "PGVectoRs", collection_name: str, enable_vector_index: bool) -> None:
```

Comment:
should this have a default option? Is there a reason to set this to false?

Reply:
okay, thanks review again, i will later to add more feature for this , when this pr ready , will invite you to review again